integrations/access/common/teleport/client.go

```diff
@@ -43,4 +43,5 @@ type Client interface {
 	ListAccessLists(context.Context, int, string) ([]*accesslist.AccessList, string, error)
 	ListAccessMonitoringRulesWithFilter(ctx context.Context, req *accessmonitoringrulesv1.ListAccessMonitoringRulesWithFilterRequest) ([]*accessmonitoringrulesv1.AccessMonitoringRule, string, error)
 	GetAccessListOwners(ctx context.Context, accessListName string) ([]*accesslist.Owner, error)
+	GetUser(ctx context.Context, name string, withSecrets bool) (types.User, error)
```

Comment:
https://github.com/gravitational/teleport/actions/runs/14977156373/job/42072277653?pr=54449#step:5:1002

I think you'll need to update the `integrations/access/accessrequest/app_test.go` and implement `GetUser` for the mock client.